docs/en/ufs/Minio.md

```diff
@@ -1,17 +1,17 @@
 ---
 layout: global
-title: Minio
-nickname: Minio
+title: MinIO
```

Comment:
On the Minio product page they use `MinIO`. Is there a reason to change that here?

Reply:
https://min.io/product